plugins/content/course/index.js

```diff
@@ -47,6 +47,7 @@ function initialize () {
       var options = _.keys(req.body).length
       ? req.body
       : req.query;
+      var options = JSON.stringify(options);
```

Comment:
@darylhedley - actually, can you remove this bit? I think this is related to the issues with node v0.12, but we're only supporting 0.10.33 at the moment, and this workaround needs a closer look.